### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RomSmile**
DEMO LINK: https://RomSmile.github.io/react_static-list-of-todos/

### CODE REVIEW

**Review by Andrii-Pinchuk — APPROVED** [review]
It's better to add some styles to your homework tasks. This can help you understand how styles work for the nested elements inside components.

File: src/components/UserInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+type Prop = {
+  userItem: ForPreparedTodos,
+};
+
+export const UserInfo: React.FC<Prop> = ({ userItem }) => {
+  const { user } = userItem;
+
+  return (
+    <>
```
Comment (varseniuk) [inline src/components/UserInfo.tsx:12]: better to use more semantic tags instead of fragment here

File: src/react-app-env.d.ts [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+interface ForTodo {
+  userId: number;
+  id: number;
+  title: string;
+  completed: boolean;
+}
+
+type ForUsers = ForUser[];
+type ForTodos = ForTodo[];
```
Comment (varseniuk) [inline src/react-app-env.d.ts:34]:
```suggestion
```
No need in these types

File: src/react-app-env.d.ts [hunk 3]
```diff
@@ -1 +1,42 @@
...
+  phone: string;
+  website: string;
+  company: {
+    name: string;
+    catchPhrase: string;
+    bs: string;
+  };
+};
+
+interface ForTodo {
```
Comment (varseniuk) [inline src/react-app-env.d.ts:26]:
```suggestion
interface Todo {
```

File: src/react-app-env.d.ts [hunk 4]
```diff
@@ -1 +1,42 @@
...
+  userId: number;
+  id: number;
+  title: string;
+  completed: boolean;
+}
+
+type ForUsers = ForUser[];
+type ForTodos = ForTodo[];
+
+export type ForPreparedTodos = {
```
Comment (varseniuk) [inline src/react-app-env.d.ts:36]:
```suggestion
export type PreparedTodos = {
```

**Review by varseniuk — APPROVED** [review]
Good. But pay attention to namings

**Pull request closed without merging**